diff --git a/es6/polar/Pie.js b/es6/polar/Pie.js
index 0e4dc0a9fc6baa233c6903d1b5a1b8d545e32932..a17574b015a20d3bd9e06d1904ecaf9436fbd10a 100644
--- a/es6/polar/Pie.js
+++ b/es6/polar/Pie.js
@@ -542,7 +542,8 @@ _defineProperty(Pie, "getComposedData", function (_ref4) {
         startAngle: tempStartAngle,
         endAngle: tempEndAngle,
         payload: entry,
-        paddingAngle: mathSign(deltaAngle) * paddingAngle
+        paddingAngle: mathSign(deltaAngle) * paddingAngle,
+        outerRadius: entry.outerRadius ? coordinate.outerRadius * entry.outerRadius : coordinate.outerRadius
       });
       return prev;
     });
